v22.2/ui-transactions-page.md

```diff
@@ -7,22 +7,65 @@ docs_area: reference.db_console
 
 {% include {{ page.version.version }}/ui/admin-access.md %}
 
-The **Transactions** page helps you:
+The **Transactions** page offers two views on transactions: **Transaction Fingerprints**, which represents one or more completed SQL transactions and **Active Executions**,
+which represents individual transaction executions in progress.
 
-- Identify frequently [retried]({{ link_prefix }}transactions.html#transaction-retries) transactions.
-- [Troubleshoot]({{ link_prefix }}query-behavior-troubleshooting.html) high latency transactions or execution failures.
-- View transaction [details](#transaction-details-page).
+You choose a view by selecting the **Transaction Fingerprints** or **Active Executions** radio button. The selection is retained when you switch between the Statements and Transactions tabs on the SQL Activity page.
 
 {{site.data.alerts.callout_success}}
-In contrast to the [**Statements** page](ui-statements-page.html), which displays [SQL statement fingerprints](ui-statements-page.html#sql-statement-fingerprints), the **Transactions** page displays SQL statement fingerprints grouped by [transaction](transactions.html).
+In contrast to the [**Statements** page](ui-statements-page.html), which displays [SQL statement fingerprints](ui-statements-page.html#sql-statement-fingerprints), the **Transactions** page displays transaction fingerprints, that is, SQL statement fingerprints grouped by [transaction](transactions.html).
 {{site.data.alerts.end}}
 
+## Transaction Fingerprints view
+
+The **Transaction Fingerprints** view helps you:
+
+- Identify frequently [retried]({{ link_prefix }}transactions.html#transaction-retries) transactions.
+- [Troubleshoot]({{ link_prefix }}query-behavior-troubleshooting.html) high latency transactions or execution failures.
+- View transaction [details](#transaction-details-page).
+
 {% if page.cloud != true %}
-To view this page, click **SQL Activity** in the left-hand navigation of the DB Console. Click the **Transactions** tab.
+To view this page, click **SQL Activity** in the left-hand navigation of the DB Console.
 {% else %}
 To view this page, click **SQL Activity** in the left-hand navigation of the {{ site.data.products.db }} Console. Click the **Transactions** tab.
 {% endif %}
 
+Click the **Transactions** tab. The **Transaction Fingerprints** radio button is selected and the [Transactions table](#transactions-table) displays.
```

Comment:
```suggestion
When the **Transactions** tab is selected, click the **Transaction Fingerprints** radio button. The [Transactions table](#transactions-table) displays.
```

Reply:
No. When you click SQL Activity, the Statements tab and Statement Fingerprints button are selected by default. You need to explicitly select the Transaction tab first.